flex/http.py

```diff
@@ -120,10 +126,27 @@ def _normalize_python3_urllib_request(request):
     )
 
 
+def _normalize_tornado_request(request):
```

Comment:
I think I would rather expand the test runs to include a `py27, py27-with-tornado, py34, py34-with-tornado` so that there is verification that everything works with and without tornado.